src/components/ViewList.js

```diff
@@ -93,20 +106,26 @@ const ViewList = () => {
       />
       <label htmlFor="Clear" aria-label="Clear search bar"></label>
       <button onClick={handleClearClick}>Clear</button>
+
       <ul>
         {currentList.userList.length > 0 ? (
           filteredList.map((element, index) => (
             <div key={element.id}>
-              <input
-                type="checkbox"
-                name={element.itemName}
-                id={element.id}
-                value={element.itemName}
-                className="purchased"
-                onChange={handleCheck}
-                checked={element.isPurchased}
-              ></input>
-              <li> {element.itemName} </li>
+              <li>
```

Comment:
I like this nesting a lot more than what we had previously. Shayne and I had a few edits here as well (styling, mostly). When merging we should keep this nesting but move class styling from the other issue.